content/plugins/aws.md

```diff
@@ -19,22 +22,26 @@ For more information about the library, please refer to: [http://boto.readthedoc
 * Python Versions:
   * 2.7.x
 * An account with AWS
-  * Your AWS [Access Keys](http://docs.aws.amazon.com/AWSSecurityCredentials/1.0/AboutAWSCredentials.html#)
 
 
 # Compatibility
 
-The AWS EC2 plugin currently only supports classic EC2.
-
 The AWS plugin uses the following client libraries:
 
-  * [Boto client](https://github.com/boto/boto) - 2.34.0
+  * [Boto client](https://github.com/boto/boto) - 2.38.0
```

Comment:
No point of list if that's a single item.

The AWS plugin uses Boto 2.38.0 client...